bfpart

```diff
@@ -46,7 +46,7 @@ default_cfg()
 
 bfcfg_scan()
 {
-  local disk_name part_size part_type part_persist part_mount
+  local disk_name part_size part_type part_persist part_mount part_type_tmp
```

Comment:
Where is the part-type_tmp used?

Reply:
Thank you for catching this! I forgot to remove this variable from debugging.